test/portworx/testspec/storagecluster_all_compenents_enabled.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   labels:
     heritage: "Helm"
     release: "my-release"
-    chart: "portworx-3.1.0"
+    chart: "portworx-3.1.2"
```

Comment:
Do these UTs have to refer to the corresponding portworx version as the chart changes?

Reply:
yes @diptiranjanpx. As we have comparing whole file with generated storageCluster yaml it needs to be change for each chart changes. 
I will change comparison logic in Improvement PR.